app/processors/base_processor.rb

```diff
@@ -7,6 +7,7 @@
 require 'fixer_constants'
 require 'system_information'
 require 'service_options'
+require 'audio_monster'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.